packages/create-spectacle/package.json

```diff
@@ -35,7 +35,24 @@
     "lint": "wireit",
     "lint:fix": "wireit",
     "prettier": "wireit",
-    "prettier:fix": "wireit"
+    "prettier:fix": "wireit",
+    "examples:clean": "rimraf .examples",
+    "examples:test": "nps jest",
+    "examples:jsx:clean": "rimraf .examples/jsx",
```

Comment:
I'm not totally in love with this verbose enumeration of tasks. But it does make scripting in CI much simpler... Open to other suggestions!